dazel.py

```diff
@@ -16,6 +16,7 @@
 DEFAULT_RUN_COMMAND = "/bin/bash"
 DEFAULT_DOCKER_COMMAND = "docker"
 DEFAULT_LOCAL_DOCKERFILE = "Dockerfile.dazel"
+DEFAULT_BUILD_DIRECTORY = os.getcwd()
```

Comment:
This is a backward-incompatible change. There might be people relying on `DAZEL_DIRECTORY` to set their context directly.

Reply:
I pushed a change that should address your concern, it now defaults to directory.